src/Vts.Test/Vts.Test.csproj

```diff
@@ -42,8 +42,8 @@
     <EmbeddedResource Include="Resources\sourcetest\inputAOfXAndYAndZ.txt" />
   </ItemGroup>
   <ItemGroup>
-    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="17.11.1" />
-    <PackageReference Include="NSubstitute" Version="5.1.0" />
+    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="17.13.0" />
+    <PackageReference Include="NSubstitute" Version="5.3.0" />
     <PackageReference Include="NUnit" Version="3.14.0" />
     <PackageReference Include="NUnit3TestAdapter" Version="4.5.0">

```

Comment:
I will update these 2 packages in a separate branch after master is merged into all the branches.

Reply:
When will you update NUnit?

Reply:
After we've merged master into the branches so it doesn't cause many errors. Then we can update any statements that do not have Assert.That using the regular expressions.

Reply:
I'm okay with you merging into master anytime you say.